Websites/bugs.webkit.org/Bugzilla/BugMail.pm

```diff
@@ -467,9 +467,7 @@ sub _generate_bugmail {
     }
 
     my $email = Bugzilla::MIME->new($msg_header);
-    if (scalar(@parts) == 1) {
```

Comment:
It would be helpful to mark this as our change, for easier upgrading in the future.

Have you verified that the received email is still readable in this case?

Reply:
Added comments to indicate WEBKIT CHANGES.
I have checked that there no pending email issues. WKBI is receiving mails, and I haven't seen any error reported caused by this bug.